Materials/DynamicMaterialTest/EmissiveMaterial.azsl

```diff
@@ -52,6 +52,7 @@ struct VSOutput
     float3 m_shadowCoords[ViewSrg::MaxCascadeCount] : UV3;
 };
 
+#include <Atom/Features/Vertex/VertexHelper.azsli>
```

Comment:
I remember I have removed this file in the TBN changes.
It shouldn't exist in Atom. Does this azsl compile?